### DISCUSSION TITLE
Send signals (static window messages)

### DISCUSSION THREAD

**Discussion by Erquint** [winsiderss/systeminformer#1441] [Questions / Answers]
I distinctly remember using a process monitor just like this one years ago to pick a signal like `WM_CLOSE` or `WM_QUIT`, etc. to be sent to a window of a process from its context menu.

*Not to be confused with POSIX signals, which aren't natively supported in Windows, but rather the static subset of [Windows Messaging](https://learn.microsoft.com/en-us/windows/win32/winmsg/windows) now interchangeably called [Window Notifications](https://learn.microsoft.com/en-us/windows/win32/winmsg/window-notifications) and Windows Signals.*

Is this signal sending functionality part of Process Explorer and I'm missing it?
Or if it's not, and I apologize for straying off topic, could somebody point me in the direction of that GUI-enabled tool if they can recall such?

**Comment by dmex** [comment]
> pick a signal like WM_CLOSE or WM_QUIT, etc. to be sent to a window of a process from its context menu.

The View > Windows dialog?

**Reply by Erquint** [reply]
Doesn't seem like it.
And I see no way to send signals from it.
